spec/syntax.md

```diff
@@ -502,7 +498,7 @@ They are allowed in translatable text as well as in literals.
 
 ```abnf
 text-escape = "\\" / "\{" / "\}"
-literal-escape = "\\" / "\(" / "\)"
+literal-escape = "\\" / "\|"
```

Comment:
These can probably benefit from the same refactor using a named backslash and grouping the backslash outside the set of escaped characters that @gibson042 [suggested](https://github.com/unicode-org/message-format-wg/pull/347#discussion_r1120574252) in the main ABNF PR.

Reply:
Good point. I'll rebase on top of #347 once it lands.

Reply:
Maybe just needs a rebase or merge from the base branch... just the literal escape bit will need a merge conflict resolution.